src/Serilog/Core/Pipeline/SilentLogger.cs

```diff
@@ -26,11 +28,11 @@ class SilentLogger : ILogger
         {
         }
 
-        public ILogger ForContext(ILogEventEnricher enricher) => this;
+        public ILogger ForContext(ILogEventEnricher? enricher) => this;
```

Comment:
I don't think `enricher` should be nullable, here (implementations receiving null just bail out instead of failing hard)

Reply:
done